src/cli.ts

```diff
@@ -28,6 +28,24 @@ Publish package versions from one registry to another.`)
     boolean: true,
     default: false
   })
+  .option('latest-only', {
+    demand: false,
+    describe: 'Only syncs the latest dist-tag version',
+    boolean: true,
+    default: false
+  })
+  .option('latest-majors', {
+    demand: false,
+    describe: 'Only syncs the latest majors',
+    boolean: true,
+    default: false
+  })
+  .option('repository', {
+    demand: false,
+    describe: 'Override the repository field in the package.json',
+    type: 'string',
```

Comment:
Codacy has a fix for the issue: [Replace `····type:·'string'` with `······type:·"string"`](https://app.codacy.com/gh/joebowbeer/regsync/pullRequest?prid=9263163)
```suggestion
      type: "string",
```